apps/strapi-cms/src/api/news-item/content-types/news-item/schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "kind": "collectionType",
+  "collectionName": "news_items",
+  "info": {
+    "singularName": "news-item",
+    "pluralName": "news-items",
+    "displayName": "NewsItem",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "title": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": true
+    },
+    "link": {
+      "type": "component",
+      "repeatable": false,
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "component": "common.links",
+      "required": true
+    },
```

Comment:
For now it is fine using link, but in future this should be a dynamic zone where use can use a link or choose a page or product and strapi will provide the correct link. Maybe is better to add an issue to backlog about this.

Reply:
Done: https://pagopa.atlassian.net/browse/DEV-1458